composer.json

```diff
@@ -12,9 +12,6 @@
     }
   },
   "config": {
-    "platform": {
-      "php": "7.2"
```

Comment:
Is there a reason for removing (as opposed to raising it to 7.3)? When it's unlocked, it's more prone to crossed wires in `composer.lock`, no?

Reply:
@totten if we had to have one I would think 7.4 or 8.0 makes more sense on a development repo these days but I guess I am not aware of any specific project here that would have issue with this being removed, Generally it is better practice to not be overriding platform I think